src/Uno.UWP/Security/Authentication/Web/WebAuthenticationBrokerProvider.cs

```diff
@@ -42,7 +48,12 @@ public Uri GetCurrentApplicationCallbackUri()
 				this.Log().Warn(message);
 			}
 
+#if __ANDROID__
+
```

Comment:
Please add a comment with explanation why the trim is needed

Reply:
#6485
Do you mean comment from some Android docs?

Reply:
Either that or just a note that the trim is necessary because otherwise it fails. Or maybe even split it like this:

```c#
var uri = new Uri(schemes[0] + WinRTFeatureConfiguration.WebAuthenticationBroker.DefaultCallbackPath);
#if __ANDROID__
uri = uri.TrimEndUriSlash()
#endif
```

To avoid duplication